assemble/bin/accumulo-cluster

```diff
@@ -299,9 +299,9 @@ function stop_all() {
 
 function stop_here() {
   # Determine hostname without errors to user
-  hosts_to_check=($(hostname -a 2> /dev/null | head -1) $(hostname -f))
+  hosts_to_check="$(hostname -a 2> /dev/null | head -1) $(hostname -f)"
```

Comment:
This is wrong. It should be assigning as an array, not a flat string. To properly quote the elements of the array, do:

```suggestion
  hosts_to_check=("$(hostname -a 2> /dev/null | head -1)" "$(hostname -f)")
```